docs/appendix/known-limitations.md

```diff
@@ -124,7 +123,7 @@ will be expanding the SDK rapidly over the next few weeks and months, but for th
 you encounter a missing API that you need, let us know, so we can ensure it is prioritized above
 APIs which are lesser used.
 
-### Rust/Miden FFI (Foreign Function Interface) and Interop
+### Rust/Miden FFI (foreign function interface) and interop
```

Comment:
Shouldn't "foreign function interface" remain capitalized here, to emphasize the correlation to the initials of the "FFI" acronym?

Reply:
Not usually: https://en.wikipedia.org/wiki/Foreign_function_interface